packages/dds/tree/src/test/feature-libraries/modular-schema/basicRebasers.ts

```diff
@@ -32,8 +31,7 @@ export function lastWriteWinsRebaser<TChange>(data: {
 	noop: TChange;
```

Comment:
`noop` doesn't seem necessary anymore

Reply:
I don't see what this has to do with my changes.

Reply:
`noop` was only needed because of compose. Now it's not needed at all.
Ah wait, it is needed for the return, please ignore.